selfservice/strategy/code/code_sender.go

```diff
@@ -0,0 +1,219 @@
+// Copyright © 2022 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package code
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/gofrs/uuid"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/pkg/errors"
+
+	"github.com/ory/herodot"
+	"github.com/ory/kratos/courier/template/email"
+
+	"github.com/ory/x/errorsx"
+	"github.com/ory/x/httpx"
+	"github.com/ory/x/sqlcon"
+	"github.com/ory/x/stringsx"
+	"github.com/ory/x/urlx"
+
+	"github.com/ory/kratos/courier"
+	"github.com/ory/kratos/driver/config"
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/selfservice/flow/recovery"
+	"github.com/ory/kratos/selfservice/flow/verification"
+	"github.com/ory/kratos/x"
+)
+
+type (
+	senderDependencies interface {
+		courier.Provider
+		courier.ConfigProvider
+
+		identity.PoolProvider
+		identity.ManagementProvider
+		identity.PrivilegedPoolProvider
+		x.LoggingProvider
+		config.Provider
+
+		RecoveryCodePersistenceProvider
+		VerificationCodePersistenceProvider
+
+		HTTPClient(ctx context.Context, opts ...httpx.ResilientOptions) *retryablehttp.Client
+	}
+	CodeSenderProvider interface {
+		CodeSender() *CodeSender
+	}
+
+	CodeSender struct {
```

Comment:
Given that the package is `code`, can we change this to `Sender`, such that other packages will see it as `code.Sender`?

Reply:
Initially, I wanted to make sure to not get it confused with `link.Sender`. But now it's clear which sender is meant.